metadata/beam_configuration.xml

```diff
@@ -155,7 +155,7 @@
             <attribute-group group-id="Beam Credentials">
                 <display-name xml:lang="x-default">Beam Impact Widget Credentials</display-name>
                 <attribute attribute-id="beamStoreId"/>
-                <attribute attribute-id="beamLang"/>
```

Comment:
`question` Was the intention to make the language ALWAYS inherit from the page, and fall back to English if it's not supported? I think keeping `beamLang` could allow a partner to specify a fallback other than English, but if that's not wanted, this seems fine.